opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableLine.js

```diff
@@ -116,16 +115,18 @@ const StixCyberObservableLineComponent = (props) => {
             >
               {nsdt(node.created_at)}
             </div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.creator.width }}
+            >
+              {t(node.creator.name)}
+            </div>
             <div
               className={classes.bodyItem}
               style={{ width: dataColumns.objectMarking.width }}
             >
               <ItemMarkings
```

Comment:
I have a problem with the display of this property
![image](https://user-images.githubusercontent.com/53513584/207545444-4bec5ab9-480f-42fe-bc67-3c50729f11e0.png)

Reply:
Done !